AssetRipper.Library/Exporters/AudioMixers/AudioMixerExporter.cs

```diff
@@ -0,0 +1,37 @@
+using AssetRipper.Core.Classes.Misc;
+using AssetRipper.Core.Interfaces;
+using AssetRipper.Core.Parser.Files.SerializedFiles;
+using AssetRipper.Core.Project.Collections;
+using AssetRipper.Core.Project.Exporters;
+using AssetRipper.Library.Configuration;
+using AssetRipper.Library.Exporters.Textures;
+using AssetRipper.SourceGenerated.Classes.ClassID_240;
+using AssetRipper.SourceGenerated.Classes.ClassID_241;
+using AssetRipper.SourceGenerated.Classes.ClassID_243;
+using AssetRipper.SourceGenerated.Classes.ClassID_245;
+using AssetRipper.SourceGenerated.Classes.ClassID_28;
+
+namespace AssetRipper.Library.Exporters.AudioMixers
+{
+	public class AudioMixerExporter : YamlExporterBase
+	{
+		public override bool IsHandle(IUnityObjectBase asset)
+		{
+			return asset is IAudioMixerController or IAudioMixerGroupController or IAudioMixerSnapshotController;
+		}
+		
+		public override IExportCollection CreateCollection(VirtualSerializedFile virtualFile, IUnityObjectBase asset)
+		{
+			IAudioMixer? audioMixer = asset switch
+			{
+				IAudioMixerController mixer => mixer,
+				IAudioMixerGroupController group => group.AudioMixer_C243.FindAsset(group.SerializedFile),
+				IAudioMixerSnapshotController snapshot => snapshot.AudioMixer_C245.FindAsset(snapshot.SerializedFile),
+				_ => null,
+			};
+			return audioMixer is IAudioMixerController audioMixerController
+				? new AudioMixerExportCollection(this, virtualFile, audioMixerController)
+				: new FailExportCollection(this, asset);
```

Comment:
If there's a reason why those assets should not be exported, that should be documented in the code. Otherwise, this check should be in `IsHandle` instead so that a different exporter can handle the asset in the situation where `FailExportCollection` would be used.

Reply:
I'm still a little hesitant about this because your comment didn't offer justification for not exporting failed assets with the default yaml exporter. However, I'm willing to let it go.